models/pancakeswap/bnb/pancakeswap_v2_bnb_amm_trades.sql

```diff
@@ -1,11 +1,11 @@
-{{ config(
+{{ config(tags=['dunesql'],
     schema = 'pancakeswap_v2_bnb',
     alias = alias('amm_trades'),
     partition_by = ['block_date'],
```

Comment:
until we finalize strategy here: https://dune.height.app/T-31748

we've been changing partitions on migrated spells to `block_month` to grow the partition sizes.

can you add a new column in the final select, right below `block_date`, where we trunc to month and output `block_month`, then apply to partition?

Reply:
this will apply to all materialized spells in here that are partitioned